lib/texter.rb

```diff
@@ -0,0 +1,19 @@
+require "twilio-ruby"
+
+class Texter
+  def initialize
+    account_sid = ENV["SID"]
+    auth_token = ENV["AUTH_TOKEN"]
+    @client = Twilio::REST::Client.new(account_sid, auth_token)
+
+    @from = ENV["TWILIO_NUMBER"]
```

Comment:
AH! I finally understand this. Spent all morning trying to figure out the implementation, and this makes much more sense!